### PR TITLE
Unique Numbers in Python

### PR DESCRIPTION
**Pull request by Jahnavi-Jonnalagadda**
**Fixes issue:** #4707 

**Changes:** Added Unique Numbers in Python

### CODE REVIEW

File: segregate_pos_neg_nos.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Implemented using Bubble Sort
+
+ar = list(map(int, input("Enter the elements: ").split()))
```
Comment (arnavb) [inline segregate_pos_neg_nos.py]: Can you move both these files to their own folders in the unclassified algorithms directory.

File: segregate_pos_neg_nos.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+n = len(ar)
+for i in range(n):
+    swapped = False
+    for j in range(0, n - i - 1):
+        if ar[j] > ar[j + 1]:
+            ar[j], ar[j + 1] = ar[j + 1], ar[j]
+            swapped = True
+    if swapped == False:
```
Comment (arnavb) [inline segregate_pos_neg_nos.py]:
```suggestion
    if not swapped:
```

File: segregate_pos_neg_nos.py (hunk 1 above)
Comment (Jahnavi-Jonnalagadda) [reply segregate_pos_neg_nos.py]: Should I put them in src

Comment (arnavb) [reply segregate_pos_neg_nos.py]: Yes.

Comment (Jahnavi-Jonnalagadda) [reply segregate_pos_neg_nos.py]: I put it in unclassified algorithms. How do I change them to their respective directories?

**Comment by Jahnavi-Jonnalagadda** [comment]
I put the file in unclassified directory but not in src. How do I change it?

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

File: segregate_pos_neg_nos.py (hunk 1 above)
Comment (Jahnavi-Jonnalagadda) [reply segregate_pos_neg_nos.py]: Made the changes, please review.

**Pull request merged**